docs/source/federated-types/sharing-types.mdx

```diff
@@ -0,0 +1,404 @@
+---
+title: Sharing types and fields
+sidebar_title: Sharing types and fields
+description: Sharing types and fields across multiple subgraphs
+---
+
+import { CodeColumns } from 'gatsby-theme-apollo-docs';
+
+> ⚠️ The field sharing model has changed significantly in Federation 2. For a summary of these changes, [see what's new](/federation-2/new-in-federation-2/#type-and-field-sharing). 
+>
+>For more information about using value types in Federation 1, you can view the [previous version of this article here](/federation/value-types/).
+
+In a federated graph, it's common to want to reuse a GraphQL type in multiple [subgraphs](./subgraphs/). 
+
+For example, suppose you want to define and reuse a generic `Position` type in different subgraphs:
+
+```graphql
+type Position {
+  x: Int!
+  y: Int!
+}
+```
+
+This article describes how to share a [GraphQL Object type's](https://www.apollographql.com/docs/apollo-server/schema/schema/#object-types) fields in a federated graph, allowing multiple subgraphs to **redefine** and **resolve** those shared fields. 
+
+## Using `@shareable`
+
+The `@shareable` directive enables multiple subgraphs to resolve a particular field, or set of fields, on an Object type across multiple subgraphs.
+
+The `@shareable` directive is new in Federation 2. To enable it, you first need to add the following snippet to the top of **all of your subgraph files** to [opt-in to Federation 2](../federation-2/moving-to-federation-2/#opt-in-to-federation-2):
+
+```graphql
+extend schema
+    @link(url: "https://specs.apollo.dev/federation/v2.0",
+       import: ["@key", "@requires", "@shareable"])
+```
+
+Now you can apply the `@shareable` directive to the object types that you want to share across multiple subgraphs:
+
+<CodeColumns>
+
+```graphql{2}
+# Subgraph A
+type Position @shareable {
+  x: Int!
+  y: Int!
+}
+```
+
+```graphql{2}
+# Subgraph B
+type Position @shareable {
+  x: Int!
+  y: Int!
+}
+```
+<></>
+
+</CodeColumns>
+
+> If a type or field is marked `@shareable` in _any_ subgraph, it must be marked `@shareable` or [`@external`](../federation-spec/#external) in _every_ subgraph that defines it. Otherwise, composition fails.
+
+The `@shareable` directive enables multiple subgraphs to resolve shared fields. So in the example above, both subgraphs A _and_ B can resolve the `Position.x` and `Position.y` fields.
+
+> ⚠️ **Important:** If multiple subgraphs can resolve a field, **make sure each subgraph's resolver for that field behaves identically.** Otherwise, queries might return inconsistent results depending on which subgraph resolves the field.
+
+Adding the `@shareable` directive to a type is a shortcut to sharing every field within that type. You can also share individual fields, like so:
```

Comment:
"Applying the `@shareable` directive to a type is equivalent to applying it to every _field_ of that type."

"You can also mark individual fields as `@shareable`, like so:"

Just slightly more distinguishing between the notion of being "shareable" and actually being "shared"